Cargo.toml

```diff
@@ -6,3 +6,12 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+clap = "4.3.17"
+derive_builder = "0.12.0"
+regex = "1.9.1"
+tree-sitter = "0.20.10"
+tree-sitter-grep = { git = "https://github.com/helixbass/tree-sitter-grep", rev = "3d4682c" }
```

Comment:
The dependency here is on the proof-of-concept `run-context` branch plus a couple of [tweaks](https://github.com/helixbass/tree-sitter-grep/pull/63)